### PR TITLE
CLDR-15327 add coverage property generation

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15327

The property file is generated while doing the coverage chart. Also cleaned up the chart (removing old comments, unused code)

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/properties/coverageLevels.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...doi ;	basicdsb ;	modernel ;	modernen ;	modernes ;	modernet ;	moderneu ;	modernfa ;	modernff_Adlm ;	moderatefi ;	modernfil ;	modernfo ;	moderatefr ;	modernga ;	moderngd ;	moderngl ;	moderngu ;	modernha ;	modernhe ;	modernhi ;	modernhi_Latn ;	modernhr ;	modernhsb ;	modernhu ;	modernhy ;	modernia ;	basicid ;	modernig ;	moderateis ;	modernit ;	modernja ;	modernjv ;	modernka ;	modernkea ;	basickgp ;	moderatekk ;	modernkm ;	modernkn ;	modernko ;	modernkok ;	modernks ;	basicks_Deva ;	basicky ;	modernlo ;	modernlt ;	modernlv ;	modernmai ;	basicmi ;	basicmk ;	modernml ;	modernmn ;	modernmni ;	basicmr ;	modernms ;	modernmy ;	modernne ;	modernnl ;	modernnn ;	modernno ;	modernor ;	modernpa ;	modernpcm ;	modernpl ;	modernps ;	modernpt ;	modernqu ;	moderaterm ;	basicro ;	modernru ;	modernsa ;	basicsat ;	basicsc ;	moderatesd ;	moderatesd_Deva ;	basicsi ;	modernsk ;	modernsl ;	modernso ;	modernsq ;	moderatesr ;	modernsu ;	basicsv ;	modernsw ;	modernta ;	modernte ;	moderntg ;	basicth ;	moderntk ;	modernto ;	basictr ;	moderntt ;	basicuk ;	modernur ;	modernuz ;	modernuz_Cyrl ;	basicvi ;	modernwo ;	basicyo ;	moderateyrl ;	moderateyue ;	modernyue_Hans ;	basiczh ;	modernzh_Hant ;	modernzu ;	moderate#EOF
```
Comment (srl295) [inline common/properties/coverageLevels.txt:2]: Looks wrong, new lines missing?

**Review by srl295 — CHANGES_REQUESTED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java [hunk 2]
```diff
@@ -601,18 +568,18 @@ public static void showCoverage(Anchors anchors, Matcher matcher, Set<String> lo
                     languageToRegion.put(ltc.transform(ltp.getLanguageScript()), country);
                 }
             }
+            languageToRegion = ImmutableMultimap.copyOf(languageToRegion);
 
             fixCommonLocales();
 
-            System.out.println(Joiner.on("\n").join(languageToRegion.keyValuesSet()));
+            System.out.println(Joiner.on("\n").join(languageToRegion.asMap().entrySet()));
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:575]: i see this is an *output* file. Are you on a a platform that expects `\r\n`? 

```suggestion
            System.out.println(Joiner.on(System.lineSeparator()).join(languageToRegion.asMap().entrySet()));
```

**Review by srl295 — CHANGES_REQUESTED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java [hunk 3]
```diff
@@ -124,13 +108,22 @@
             + "\tCode"
             + "\tST Link"
             ;
+
+    private static final String PROPERTIES_HEADER =
+        "#Data file\tproviding the Coverage Level of locales at Basic or Above."
+            + "#For more info,\tsee the Locale Coverage Chart for this version."
```
[the remark this answers is not in the document]
Comment (srl295) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:114]: wonder if it should have the current GEN_VERSION also

File: common/properties/coverageLevels.txt (hunk 1 above)
Comment (macchiati) [reply common/properties/coverageLevels.txt:2]: whoops! Did some code cleanup at the end, and missed this.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java (hunk 2 above)
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:575]: No, it expects "\n"; I think this is unrelated.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java [hunk 4]
```diff
@@ -960,38 +916,90 @@ public static void showCoverage(Anchors anchors, Matcher matcher, Set<String> lo
...
                     .addCell(coreMissingString)
-                    .addCell(visibleImputed)
-                    .addCell(computed == cldrLocaleLevelGoal ? " ≡" : " ≠")
                     .finishRow();
 
+                    // now write properties file line
+
+                    if (computed != Level.UNDETERMINED) {
+                        propertiesCoverage.println(locale
+                            + " ;\t" + visibleComputed);
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:961]: much better!

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java [hunk 5]
```diff
@@ -124,13 +108,30 @@
             + "\tCode"
             + "\tST Link"
             ;
+
+    private static final String PROPERTIES_HEADER =
+        // Extra tabs are for github table formatting
+        "#\tcoverageLevels.txt\t\n"
+        + "#\tCopyright © 2022 Unicode, Inc.\n"
+        + "#\tCLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)\n"
+        + "#\tFor terms of use, see http://www.unicode.org/copyright.html\n"
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:117]:
```suggestion
        + CldrUtility.getCopyrightString("#\t")
```

**Review by srl295 — APPROVED** [review]
LGTM

**Review by srl295 — APPROVED** [review]
rslgtm

**Pull request merged**